components/UploadForm.vue

```diff
@@ -258,6 +243,7 @@ const onFileUpload = async (event: DragEvent) => {
       }
     }
   } finally {
+    await estimateArweaveFee()
```

Comment:
Should this be in try block?

Reply:
[🥅 Add error handling for estimateArweaveFee](https://github.com/likecoin/nft-book-press/pull/290/commits/bff4d742331d16a2c20097ba2580f89bea197fcb)